pkg/apis/core/v1/zz_generated.validations.go

```diff
@@ -35,120 +35,4614 @@ func init() { localSchemeBuilder.Register(RegisterValidations) }
 // RegisterValidations adds validation functions to the given scheme.
 // Public to allow building arbitrary schemes.
 func RegisterValidations(scheme *runtime.Scheme) error {
-	scheme.AddValidationFunc(&v1.Pod{}, func(obj, oldObj interface{}, subresources ...string) field.ErrorList {
+	scheme.AddValidationFunc(new(v1.Binding), func(obj, oldObj interface{}, subresources ...string) field.ErrorList {
```

Comment:
Not to self: This registers validations for all types now, not just root types.

Reply:
Oh, that was not the intent, that seems like a bug.  I wanted to emit code for all types but only register root types (types with...)